### ISSUE TITLE
canary releases have unexpected version numbers

### ISSUE THREAD

**Issue by geoknee** [statechannels/statechannels#3165]
**Describe the bug :bug:**
Canary releases seem to have out-of-date or old version numbers

**To Reproduce**
Trigger a canary release as so https://github.com/statechannels/statechannels/runs/1728720216?check_suite_focus=true
Note that the commit pulled by the action has the server wallet at `1.17.0` initially. 
But lerna prints ` - @statechannels/server-wallet => 1.14.2-next.f71f387dc.331+f71f387dc`

**Expected behavior**
A version number of either 1.17.something or perhaps 1.18.something

**Comment by wunnaii** [comment]
Blocked by #3170 ?

**Comment by geoknee** [comment]
I don't think so, actually. I don't think there are any blockers for this right now.
